File: MATLAB/crop.m
img=imread("temple.jpg")
gray=rgb2gray(img);
gray=im2double(gray);
mask=gray>125/255
result=img;
result(repmat(~mask,[1,1,3]))=0;
subplot(2,2,1)

imshow(result)
subplot(2,2,2)

imshow(gray)
subplot(2,2,3)

imshow(img)

File: OPENCV/crop.py
import cv2 as cv
import numpy as np
img = cv.imread('asset/temple.jpg')
q=cv.cvtColor(img,cv.COLOR_BGR2RGB)
height,width,channels=q.shape
crop=img[0:height-250,0:width-1000,:]
cv.imshow("img",crop)
cv.waitKey(0)
cv.destroyAllWindows()
